src/client/utils/getQuoteDetails.ts

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { QuoteDetails } from '../data/graphql'
+import { OfferQuote } from '../pages/OfferNew/types'
+import {
+  typeOfResidenceTextKeys,
+  quoteDetailsHasAddress,
+  getHouseholdSize,
+  HomeInsuranceTypeOfContract,
+} from '../pages/OfferNew/utils'
+import { TextKeyMap } from './textKeys'
+import { formatPostalNumber } from './postalNumbers'
+
+type DetailsGroup = ReadonlyArray<{
+  key: string
+  label: React.ReactNode
+  value: React.ReactNode
+}>
+
+function getHouseSummaryDetailsMaybe(
```

Comment:
I would consider put these functions into separate modules

Reply:
I left them in the same file for now, as they are all used in the getQuoteDetails function to summarize the quote... We will have to write new versions of these for quotecart, so I am not sure where these will be used in the future? To me it feels good to have them gathered for now 🤔